server/services/startup.ts

```diff
@@ -1,28 +1,40 @@
 import { MongoInternals } from 'meteor/mongo';
 
+import { AnalyticsService } from './analytics/service';
 import { api } from '../sdk/api';
-import { Authorization } from './authorization/service';
+import { AuthorizationLivechat } from '../../app/livechat/server/roomAccessValidator.internalService';
+import { AuthorizationTokenpass } from '../../app/tokenpass/server/roomAccessValidator.internalService';
 import { BannerService } from './banner/service';
+import { LDAPService } from './ldap/service';
+import { MediaService } from './image/service';
 import { MeteorService } from './meteor/service';
 import { NPSService } from './nps/service';
 import { RoomService } from './room/service';
+import { SAUMonitorService } from './sauMonitor/service';
 import { TeamService } from './team/service';
 import { UiKitCoreApp } from './uikit-core-app/service';
-import { MediaService } from './image/service';
-import { AnalyticsService } from './analytics/service';
-import { LDAPService } from './ldap/service';
-import { SAUMonitorService } from './sauMonitor/service';
 
 const { db } = MongoInternals.defaultRemoteCollectionDriver().mongo;
 
-api.registerService(new Authorization(db));
+api.registerService(new AnalyticsService(db));
+api.registerService(new AuthorizationLivechat());
+api.registerService(new AuthorizationTokenpass());
 api.registerService(new BannerService(db));
+api.registerService(new LDAPService());
+api.registerService(new MediaService());
 api.registerService(new MeteorService());
-api.registerService(new UiKitCoreApp());
 api.registerService(new NPSService(db));
 api.registerService(new RoomService(db));
-api.registerService(new TeamService(db));
-api.registerService(new MediaService());
-api.registerService(new AnalyticsService(db));
-api.registerService(new LDAPService());
 api.registerService(new SAUMonitorService());
+api.registerService(new TeamService(db));
+api.registerService(new UiKitCoreApp());
+
+// if TRANSPORTER env var it means the process is running in micro services mode
+// in that case we don't need to register services that will run separately
+if (!process.env.TRANSPORTER?.match(/^(?:nats|TCP)/)) {
+	(async (): Promise<void> => {
+		const { Authorization } = await import('./authorization/service');
```

Comment:
I'm curious 🤔 can this be a separate function?
Like:
```
api.importAndRegister(path: string, ...deps: any[])
```

That way, we won't have to import things here and the iife won't grow too big 😬

Reply:
hmm.. interesting 🤔  I'll take a look and see if it works

Reply:
this is definitely a good idea.. moleculer has the same feature to create/register services by using `broker.loadService('./file.js')` .. 

this is not something I'll provide on this PR, but I'll take notes to look at this in the future 👍 

thx again for bringing up this idea @KevLehman